test/HurricaneLaura/coastal_domain_testing.yaml

```diff
@@ -0,0 +1,73 @@
+---
+#initial input parameters
+log_parameters:
+    showtiming: true
+    log_level: DEBUG
+network_topology_parameters:
+    supernetwork_parameters:
+        title_string: Lower Colorado, open-loop
+        geo_file_path: /glade/scratch/adamw/CONUS_1yr/domain/RouteLink_NWMv2.1.nc

```

Comment:
Here are some more hard-coded glade paths

Reply:
The yaml file is removed.